spec/index.html

```diff
@@ -11,35 +11,31 @@
     <script class="remove">
       // All config options at https://respec.org/docs/
       var respecConfig = {
-        group:                "rdf-star" ,
-        localBiblio:          localBibliography,
         specStatus:           "ED",
         edDraftURI:           "https://w3c.github.io/sparql-entailment/spec/",
         testSuiteURI:         "https://w3c.github.io/rdf-tests/",
         shortName:            "sparql12-entailment",
         copyrightStart:       "2012",
 
-        github:               "https://github.com/w3c/sparql-entailment",
-        wgPublicList:         "public-rdf-star-wg",
-
-//        implementationReportURI: "https://w3c.github.io/sparql12-entailment/reports/",
-//        errata:               "https://w3c.github.io/sparql12-entailment/errata/",
-
-        previousPublishDate:  "2013-03-21",
-        prevRecURI:           "https://www.w3.org/TR/2013/REC-sparql11-entailment-20130321",
-        prevRecShortname:     "sparql11-entailment",
         previousPublishDate:  "2013-03-21",
         previousMaturity:     "REC",
-        
-         editors: [
+        prevRecShortname:     "sparql11-entailment",
+
+        editors: [
            { name: "Peter Patel-Schneider", w3cid: "13382"}
-         ],
+        ],
         formerEditors: [
           { name: "Birte Glimm" },
           { name: "Chimezie Ogbuji" },
         ],
 
-        //doJsonLd:     true,
+        github:               "https://github.com/w3c/sparql-entailment",
+        group:                "rdf-star" ,
+        doJsonLd:     true,
+        wgPublicList:         "public-rdf-star-wg",
+//        implementationReportURI: "https://w3c.github.io/sparql12-entailment/reports/",
+//        errata:               "https://w3c.github.io/sparql12-entailment/errata/",
+//        localBiblio:          localBibliography,
```

Comment:
I believe that this should not be commented out for the FPWD. This is what is causing 5 errors and 31 warnings in the editors draft.

Reply:
OK, I guess.